docs/production/deploying-to-cloud/gcp/setup.mdx

```diff
@@ -125,12 +125,12 @@ Here are the steps we’ll take:
 
         1. Sample commands if using previously tagged custom Docker image:
             ```bash
-            docker tag mageprod:latest [region]-docker.pkg.dev/[project_id]/[repository]/mageai:latest
```

Comment:
The mageprod image name is an example that was used in this guide. It follows along the examples in previous steps.

Reply:
Can we undo these changes please?

Reply:
I updated the docs to call out the tag: https://github.com/mage-ai/mage-ai/pull/1797.

You can undo your changes at this line.